README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+use of HTML to reinforce the meaning of content on a web page, rather than merely define its appearance. Semantic Elements describe the meaning and the content to the browser.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display: block - a line break before and after the element. display: inline - no line break before or after the element.
```

Comment:
great answer, many people have issues distinguishing the two. this is an elegant and short definition